Userland/Libraries/LibGUI/AbstractButton.h

```diff
@@ -18,6 +18,8 @@ class AbstractButton : public Widget {
     virtual ~AbstractButton() override;
 
     Function<void(bool)> on_checked;
+    Function<void()> on_enter;
+    Function<void()> on_leave;
```

Comment:
This feels really awkward since buttons generally don't have enter/leave behaviors.

Reply:
Removed since it's no longer needed along with the tooltip change.